src/libraries/System.Management/src/System/Management/WMIGenerator.cs

```diff
@@ -5927,28 +5927,28 @@ private CodeExpression CreateObjectForProperty(string strType, CodeExpression pa
         /// Internal function used to create code to convert DateTime or ManagementPath to String
         /// convert a expression. Used in adding code for Property Set for DateTime and Reference properties
         /// </summary>
-        private CodeExpression ConvertPropertyToString(string strType, CodeExpression beginingExpression)
+        private CodeExpression ConvertPropertyToString(string strType, CodeExpression beginningExpression)
```

Comment:
I think this is okay as it's not a part of public API, but pointing to it just in case.